circle.yml

```diff
@@ -5,19 +5,18 @@ machine:
     version: oraclejdk8
   environment:
     GITHUB_PAGES_RUBY_VERSION: $(curl -s https://raw.githubusercontent.com/github/pages-gem/master/.ruby-version | cut -d '-' -f 1)
+    NOKOGIRI_USE_SYSTEM_LIBRARIES: true
```

Comment:
Please document reason for addition in an inline comment (I guess it's coming from using https://github.com/gjtorikian/html-proofer, we should remove that line if we stop using that tool).